orchagent/portsorch.cpp

```diff
@@ -4007,6 +4158,12 @@ bool PortsOrch::addLagMember(Port &lag, Port &port, bool enableForwarding)
     LagMemberUpdate update = { lag, port, true };
     notify(SUBJECT_TYPE_LAG_MEMBER_CHANGE, static_cast<void *>(&update));
 
+    if (gMySwitchType == "voq")
```

Comment:
The syncing function voqSyncAddLagMember() has check for this. If the lag's switch id is not equal to the local switch id, i.e., if the lag is remote, it will not be sync-ed.